DevCycle.SDK.Server.Common/API/DevCycleProvider.cs

```diff
@@ -66,6 +66,20 @@ public override async Task<ResolutionDetails<Value>> ResolveStructureValueAsync(
             return new ResolutionDetails<Value>(flagKey, openFeatureValue, ErrorType.None,
                 variable.IsDefaulted ? Reason.Default : Reason.TargetingMatch);
         }
+        
+        public override Task ShutdownAsync(CancellationToken cancellationToken = new CancellationToken())
```

Comment:
is there an initialize method?

Reply:
not overloaded - we're leaving it as the default since the init is handled by the constructor

Reply:
don't we ususally use the init method to tell OF when we are initialized?

Reply:
Same as with other providers - our provider is "immediately" available due to the defaulting nature; we don't have a good way to handle/know when WASM is fully initialized here - so we just say we're immediately available

Reply:
That doesn't exist in .NET for the provider - the other caveat is that the initialize function is async - so that wouldn't block the provider from running either way

Reply:
This is the parent class we're implementing - https://github.com/open-feature/dotnet-sdk/blob/main/src/OpenFeature/FeatureProvider.cs

Initialize is async - so it's not really useful to block when the function itself is not going to block the init. https://github.com/open-feature/dotnet-sdk/blob/main/src/OpenFeature/FeatureProvider.cs#L121

Reply:
Additionally - by implementing an await handler (the event callback setup that the Java/JS use) would force the provider into a blocking init - and not allow/use the async init of the SDK itself.